shared/message/support/Gazebo/GazeboBallLocation.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+
+package message.support.Gazebo;
+
+message GazeboBallLocation {
+
+    double x = 1;
+    double y = 2;
+    double z = 3;
+
+    double velx = 4;
+    double vely = 5;
+    double velz = 6;
+
+    string sourceName = 7;
```

Comment:
What is this variable, also why is it camel case?

Reply:
*shrugs*